honeybee_energy/lib/_loadschedules.py

```diff
@@ -9,8 +10,7 @@
 
 
 # load schedules from the default and user-supplied files
-cur_dir = os.path.dirname(__file__)
-schedule_lib = os.path.join(cur_dir, 'library', 'schedules')
+schedule_lib = os.path.join(folders.template_library_folder, 'schedules')
```

Comment:
I agree with the first point that it would be good to not rebuild it in different module each time. So I will definitely add a read-only property. I'm not so sold on the second item, though, since I think it will be a more common case to reset the folder for all of the libraries at once (materials, schedules, programtypes, constructionists) rather than setting individual libraries. But having everything use the same properties leaves this option open down the line if we find that we need it.

Reply:
I added the read-only property.